docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
Or `$dip` right above it for a more complicated one involving a `$(q.gen t.q.gen)`.